src/pages/awesome.js

```diff
@@ -0,0 +1,180 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+// import PluginDiv from "../components/plugin_div.js";
+import NavBar from "../components/navbar.js";
+import {Footer} from "../components/footer.js";
+import {default as InfiniteScroll} from "infinite-scroll";
+import $ from "jquery";
```

Comment:
Ah, I would like to write this in pure react rather than importing jquery like this. I think using a different library rather than infinte-scroll is better and I'll have a look at this a bit later.